.travis.yml

```diff
@@ -3,8 +3,6 @@ language: node_js
 addons:
   sauce_connect: true
 node_js:
- - 4 # to be removed 2018-04-30
- - 6 # to be removed 2019-04-01
  - 8 # to be removed 2019-12-31
  - 9 # to be removed 2018-06-30
```

Comment:
Probably worth removing v9 as well (end of life since 2018-06-30), and add v10 and v12 to the list, no?